packages/transitive-overlay/src/index.tsx

```diff
@@ -176,20 +254,21 @@ const TransitiveCanvasOverlay = ({
         }}
         type="symbol"
       />
-      <Layer filter={["==", "type", "place"]} id="places" type="circle" />
       <Layer
-        filter={["==", "type", "place"]}
-        id="places-labels"
-        layout={{
-          "symbol-placement": "point",
-          "text-field": ["get", "name"],
-          "text-size": 12,
-          "text-anchor": "top",
-          "text-padding": 5,
-          "text-optional": true,
-          "text-allow-overlap": false
+        filter={stopFilter}
+        id="stops-circles"
+        paint={{
+          "circle-color": "#fff",
+          "circle-radius": 5,
+          "circle-stroke-width": 2
         }}
-        paint={{ "text-translate": [0, 5] }}
+        type="circle"
+      />
+      <Layer
+        filter={stopFilter}
+        id="stops-labels"
+        layout={defaultTextLayoutParams}
```

Comment:
It's a bit distracting to see the station labels at the same size as the origin/destination labels. Is there any way to adjust the priority/min zoom level/font size/font color of these labels so they don't overpower the origin/destination labels?

Reply:
I was looking to some way to make the origin/destination bold, check it out in https://github.com/opentripplanner/otp-ui/pull/452/commits/8a5358d36319eb6f4b206b421e386386662748b4. With this, origin/destination will have the top priority, followed by transit stations then access leg places.